fcrepo-http-api/src/main/java/org/fcrepo/http/api/FedoraLdp.java

```diff
@@ -645,4 +725,25 @@ private void checkLinkForLdpResourceCreation(final String link) {
         }
     }
 
+    /**
+     * Parse the RFC-3230 Digest response header value.  Look for a
+     * sha1 checksum and return it as a urn, if missing an empty string is
+     * returned.
+     * @param digest The Digest header value
+     * @return the sha1 checksum value
+     */
+    private String parseDigestHeader(final String digest) {
+        if (digest != null) {
+            final Map<String,String> digestPairs = RFC3230_SPLITTER.split(digest);
+            for (Map.Entry<String, String> digestPair : digestPairs.entrySet()) {
+                switch(digestPair.getKey().toLowerCase()) {
```

Comment:
Oh, I'm not sure of that myself.It was an honest question. I find bit fixity confusing and depressing, like every other desperate attempt to stab the beating heart of time and lay it cold and still, so I try to stay away from it. If we have any plans to add acceptable hashes, it's okay by me, but we should have a comment here to that effect. Otherwise, yeah, maybe just the simpler conditional.